### DISCUSSION TITLE
建议将 音量快照 改为 通知栏快照

### DISCUSSION THREAD

**Discussion by jiuqianyuan** [gkd-kit/gkd#133] [Ideas]
建议将音量键快照改为在
- 通知中设置一个快照按钮  
- 或者直接设置 通知栏快捷开关，

这样既不影响音量调节，也不像悬浮窗快照一样影响显示

![45451545466](https://github.com/gkd-kit/gkd/assets/39406781/52ab9c81-2aa8-4a7a-b1a8-25b8bedef5a0)

**Comment by lisonge** [comment]
https://github.com/gkd-kit/gkd/releases
